crates/gwt/web/styles/components.css

```
@@ -203,12 +203,139 @@ body::after {
+}
+
+.update-modal__details dt {
+  margin: 0;
+  color: var(--color-text-secondary);
+  font-weight: 600;
+}
+
+.update-modal__details dd {
+  margin: 0;
+  color: var(--color-text-primary);
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)